src/modules/BootSupportCheck.rb

```diff
@@ -191,6 +194,8 @@ def check_zipl_part
         return false
       end
 
```

Comment:
According to my knowledge, ext partition can be even just "/". And I check it elsewhere and for installing gentoo they suggest to have simple "/" as ext4 and no other partitions, see https://wiki.gentoo.org/wiki/S390/Install#Initializing_and_formatting_the_DASD_disks